trafix-codec/src/decoder/decode.rs

```diff
@@ -47,15 +47,27 @@ pub enum Error {
     #[error(
         "calculated and expected checksums don't match 'calculated({calculated}) != ({expected})'"
     )]
-    ChecksumMismatch { calculated: u8, expected: u8 },
+    ChecksumMismatch {
+        /// Checksum calculated from the message.
+        calculated: u8,
+
+        /// Expected checksum as per the FIX protocols defined algorithm.
+        expected: u8,
```

Comment:
I suggest rewording this doc comment slightly: 


```suggestion
        /// Expected checksum that was extracted from the FIX message.
        expected: u8,
```